auth_token/models.py

```diff
@@ -36,6 +39,8 @@ class Token(models.Model):
     backend = models.CharField(max_length=255, null=False, blank=False)
     allowed_cookie = models.BooleanField(default=True)
     allowed_header = models.BooleanField(default=True)
+    is_authenticated = models.BooleanField(null=False, blank=False, default=False)
```

Comment:
I am wondering if this name is not too generic, wouldn't something like `is_two_factor_verified` be better?

Reply:
It is ok